cluster/log-dump/log-dump.sh

```diff
@@ -136,25 +137,25 @@ function copy-logs-from-node() {
 function save-logs() {
     local -r node_name="${1}"
     local -r dir="${2}"
-    local files="${3}"
+    local save_files="${3}"
```

Comment:
shellcheck detects the following failure. But I don't think that `files` in `save-log()` is used as an array.
shellcheck might recognize `files` in `save-log()` and `files` in `copy-logs-from-node()` are the same variable.
To avoid that, I changed variable name.
```
In ./cluster/log-dump/log-dump.sh line 139:
    local files="${3}"
          ^---^ SC2178: Variable was used as an array but is now assigned a string.
```